README.md

```diff
@@ -74,6 +74,10 @@ resource "aws_security_group_rule" "container_instance_https_egress" {
 
 ## Variables
 
+- `cluster_name` - Name of the ECS Cluster, it is optional
+- `asg_name` - Name of the ASG name for ECS Cluster, it is optional
```

Comment:
A bit of a nitpick, but could you please expand this variable name to `autoscaling_group_name`?